CDCgov/prime-reportstream
# 1138 Added support to decode email from HL7

DRAFT
This PR adds support to decode email addresses from HL7 input data.

To test:
1. Build the project using `./gradlew package` and verify the data tests run successfuly.  Note the sample-pdi CSV file contains a patient email address.

## Changes
- Add support to decode email address from HL7
- Added sample fake data that includes email address.  I did not want to change the entity provided samples, but this one we can update as we please as it is just fake data we created.
- Added unit tests

## Checklist

### Testing

### Security

### Process

## Security
*List potential security threats and mitigations in the PR*

## Fixes
- #1138

## To Be Done
